api-server/Web/Controllers/GitHubController.cs

```diff
@@ -1472,6 +1472,9 @@
                 }
             }
 
+            int pageSize = 7;
```

Comment:
<!--Using HubReview-->
thought(non-blocking): We don't use this style, but can we discuss?

Reply:
Okay!